includes/fields/class-fieldtypes-select.php

```diff
@@ -251,23 +251,35 @@ private function add_options_to_field( $options, $value, $field ) {
 			$option_data = array(
 				'label'      => $label,
 				'attributes' => array(),
+				'value'      => esc_attr( $option ),
 			);
 
 			if (
 				( ! empty( $selected ) && $option === $selected ) ||
-				in_array( $option, $value, true )
+				// phpcs:ignore
+				in_array( $option, $value )
 			) {
 				$option_data['attributes']['selected'] = 'selected';
 			}
 
 			$option_data = apply_filters( 'wpbdp_form_field_select_option', $option_data, $field );
 
-			$html .= sprintf(
-				'<option value="%3$s" class="%1$s" %2$s>%3$s</option>',
-				esc_attr( 'wpbdp-inner-field-option wpbdp-inner-field-option-' . WPBDP_Form_Field_Type::normalize_name( $option_data['label'] ) ),
-				self::html_attributes( $option_data['attributes'], array( 'value', 'class' ) ),
-				esc_html( $option_data['label'] )
-			);
+			if ( $this->is_multiple() && $field->get_association() == 'meta' ) {
```

Comment:
It seems this condition could be managed without using if and else.

Reply:
I mean like: https://github.com/Strategy11/business-directory-plugin/pull/431#discussion_r1930895065